tests/test_perf_analyzer.py

```diff
@@ -105,26 +120,16 @@ def test_run(self):
             model_path=MODEL_LOCAL_PATH,
             version=TRITON_VERSION,
             config=server_config)
-        client = PerfAnalyzer(config=self.config)
+        perf_client = PerfAnalyzer(config=self.config)
```

Comment:
There is no test here. A test requires some assert condition. Look for a way to make sure an assert is not thrown.

Reply:
I'll add some more unit tests for cases where exceptions are thrown, but if an exception is thrown, the test will fail. The `unittest` module doesn't seem to have an `assertNotRaises`. This is implicit.  I could use a `try...except` block and force fail the test if there is an exception but I don't see the need.